benchmark_runner/main/main.py

```diff
@@ -60,23 +60,26 @@
 
 
 @logger_time_stamp
-def azure_cluster_start_stop():
+def azure_cluster_start_stop_cluster():
     """
     This method starts/ stops azure cluster
     :return:
     """
-    azure_operation = AzureOperations(azure_clientid=environment_variables_dict.get('azure_clientid', ''),
+    azure_operation = AzureOperations(
+                                      azure_clientid=environment_variables_dict.get('azure_clientid', ''),
                                       azure_secret=environment_variables_dict.get('azure_secret', ''),
                                       azure_tenantid=environment_variables_dict.get('azure_tenantid', ''),
                                       azure_subscriptionid=environment_variables_dict.get('azure_subscriptionid', ''),
-                                      azure_resource_group_name=environment_variables_dict.get(
-                                          'azure_resource_group_name', ''))
+                                      azure_resource_group_name=environment_variables_dict.get('azure_resource_group_name', ''),
+                                      kubeadmin_password=environment_variables_dict.get('kubeadmin_password', '')
+                                      )
     azure_vm_name = (environment_variables_dict.get('azure_vm_name', ''))
     if azure_cluster_start:
```

Comment:
Where do azure_cluster_start and azure_cluster_stop come from?  If they're globals, it would be easier to read to pass these in.  For that matter, it would be cleaner to separate them into two functions, with a common azure_get_operation() method for the shared code.

Reply:
I put all the code in 1 method, dont think we need 2 methods for it, it will just make the code complex

Reply:
Not a big deal, OK.